package.json

```diff
@@ -65,7 +66,7 @@
     "build:digitalocean": "cd hosting/digitalocean && ./build.sh && cd -",
     "build:docker:single:multiarch": "docker buildx build --platform linux/arm64,linux/amd64 -f hosting/single/Dockerfile -t budibase:latest .",
     "build:docker:single:image": "docker build -f hosting/single/Dockerfile -t budibase:latest .",
-    "build:docker:single": "lerna run build && lerna run predocker && npm run build:docker:single:image",
+    "build:docker:single": "yarn build:docker:pre && npm run build:docker:single:image",
```

Comment:
NAB: small edge case but best to just use `npm run` instead of `yarn` in NPM scripts so they work on machines without `yarn`